httpbp/middlewares.go

```diff
@@ -44,16 +44,30 @@ func Wrap(name string, handle HandlerFunc, middlewares ...Middleware) HandlerFun
 // DefaultMiddlewareArgs provides the arguments for the default, Baseplate
 // Middlewares
 type DefaultMiddlewareArgs struct {
-	TrustHandler    HeaderTrustHandler
+	// The edgecontext implementation to use. Required.
 	EdgeContextImpl *edgecontext.Impl
+
+	// The HeaderTrustHandler to use.
+	// If empty, NeverTrustHeaders will be used instead.
+	TrustHandler HeaderTrustHandler
+
+	// The logger to be called when edgecontext parsing failed.
+	Logger log.Wrapper
 }
 
 // DefaultMiddleware returns a slice of all of the default Middleware for a
 // Baseplate HTTP server.
 func DefaultMiddleware(args DefaultMiddlewareArgs) []Middleware {
+	if args.TrustHandler == nil {
+		args.TrustHandler = NeverTrustHeaders{}
+	}
```

Comment:
Should we actually bother with this in here since we're already doing it in `InjectEdgeRequestContext`?

Reply:
This is also used by `InjectServerSpan`